webpack.config.js

```diff
@@ -122,6 +122,7 @@ const webpackConfig = {
 		],
 		alias: Object.assign(
 			{
+				'components/': '@automattic/simple-components/',
```

Comment:
Hmm, this is weird, since it's working for me...

```
lrwxr-xr-x  1 bernie  staff  23  9 Aug 10:51 node_modules/@automattic/simple-components -> ../../client/components
```